CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ title: Changelog
 
 ## Table of Contents
 
+- [3.10.0] (#3100)
```

Comment:
```suggestion
- [3.10.0](#3100)
```

otherwise
<img width="237" alt="image" src="https://github.com/user-attachments/assets/3e7adc46-856d-4f08-ab6c-48af28e86498">